.gitignore

```diff
@@ -99,7 +99,7 @@ packages/fxa-auth-server/vendor
 packages/fxa-content-server/pids
 packages/fxa-content-server/results
 packages/fxa-content-server/lib-cov
-packages/fxa-content-server/app/styles/*.css
+packages/fxa-content-server/app/styles/main.css
```

Comment:
This differs slightly than what I did to the gitignore in #13507 because I realized this is the only CSS file generated right now, and, there's a chance I might create additional CSS files when working on #13492.